pom.xml

```diff
@@ -126,6 +126,11 @@
 			<version>4.3.2.RELEASE</version>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-core</artifactId>
```

Comment:
Its better to use this starter instead of single mockito dependency
`spring-boot-starter-tomcat`